test/TesterInternal/DevTestClientConfiguration.xml

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ClientConfiguration xmlns="urn:orleans">
   <SystemStore SystemStoreType="SqlServer"
-               DataConnectionString="connection string"/>
+               DataConnectionString="connection string"
+               DeploymentId="deployment id"/>
```

Comment:
Let's fix this in a separate PR. Not sure if putting a dummy ID is what we want in this case